doc/cug.tex

```diff
@@ -2822,9 +2822,13 @@ \subsubsection{Graph Types}
             graph = "delayed_cold_corge => corge"
 
         # Repeat once at the final cycle point
-        [[[ R1/P0Y ]]]
+        [[[ R1/$ ]]]
```

Comment:
If the example list gets too unwieldy with extra additions it might be best to split it up and have it under different sub-sections.